dedibox/vps/quickstart.mdx

```diff
@@ -0,0 +1,85 @@
+---
+meta:
+  title: Dedibox VPS - Quickstart
+  description: This page explains how to get started with Dedibox VPS (Virtual Private Servers)
+content:
+  h1: Dedibox VPS - Quickstart
+  paragraph: This page explains how to get started with Dedibox VPS (Virtual Private Servers)
+tags: dedibox vps virtual-machine
+dates:
+  validation: 2024-05-15
+  posted: 2024-05-15
+categories:
+  - dedibox-vps
+---
+
+A Dedibox VPS (Virtual Private Server) operates within a larger physical infrastructure, providing directly accessible computing resources and greater control than shared hosting solutions.
+
+Dedibox VPSs are versatile, serving various purposes such as hosting websites, web applications, databases, and development environments.
+They offer a balance of performance, control, and affordability for businesses and individuals with moderate to high-traffic websites or applications.
+
+## Creating a Dedibox VPS
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- [Added a payment method](/dedibox-console/account/how-to/add-payment-method) to your account
+
+1. Navigate to your [Dedibox console](https://console.online.net) home page and select **Order** from the top navigation.
+2. Choose **Dedibox VPS** from the drop-down menu to access available offers.
+3. Select a VPS offer and proceed with the order.
+4. Specify the geographical region for deployment and confirm your choice.
+5. Choose the operating system (OS) and version for your virtual machine.
+6. Enter a hostname containing at least eight alphanumeric characters.
+7. Optionally, paste an [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/#how-to-upload-the-public-ssh-key-to-the-scaleway-interface) for authentication.
+8. Review your order details and confirm by clicking **Confirm my order**.
+    <Message type="note">
+      It may take a few minutes for your VPS to be delivered and visible in your account.
+    </Message>
+
+## Connecting to a Dedibox VPS
+
+You can manage your Dedibox VPS by accessing it through SSH.
+
+<Macro id="requirements" />
```

Comment:
I think we can get rid of the requirements for this and the next steps, seeing as it's a quickstart?